site/docs/relations/physical_relations.md

```diff
@@ -205,6 +205,23 @@ The streaming aggregate operation leverages data ordered by the grouping express
 | Per Grouping Set | A list of expression grouping that the aggregation measured should be calculated for. | Optional, defaults to 0.                |
 | Measures         | A list of one or more aggregate expressions. Aggregate expressions ordering requirements must be compatible with expected ordering. | Optional, required if no grouping sets. |
 
+## Consistent Partition Window Operation
+A consistent partition window operation is a special type of project operation where every function is a window function and all of the window functions share the same sorting and partitioning. This allows for the sort and partition to be calculated once and shared between the various function evaluations. This operator doesn't need to build hash tables as Hashing Window and rely on the sorted input as Streaming Window.
```

Comment:
```suggestion
A consistent partition window operation is a special type of project operation where every function is a window function and all of the window functions share the same sorting and partitioning. This allows for the sort and partition to be calculated once and shared between the various function evaluations.
```

I don't know that this sentence is needed.  The consumer is free to implement this however they see fit.